package.json

```diff
@@ -28,21 +28,20 @@
     "prebuild": "yarn clean",
     "build": "tsc --build",
     "watch": "tsc --build --watch",
-    "prepsuite": "node scripts/prepareSuite.js",
     "commit": "git-cz",
-    "changelog": "lerna-changelog",
     "lint:prettier": "prettier --list-different . \"!**/*.{js,ts}\" ",
     "lint:eslint": "eslint --cache --ext .js --ext .ts .",
     "lint": "yarn lint:eslint && yarn lint:prettier",
     "fix": "yarn lint:eslint --fix && yarn lint:prettier --write",
+    "prepsuite": "node scripts/prepareSuite.js",
     "pretest": "yarn build && yarn lint && yarn prepsuite",
     "test": "jest --reporters=default --reporters=jest-junit",
     "test:cli": "jest test/ --reporters=default --reporters=jest-junit --forceExit",
     "test:packages": "jest packages/ --reporters=default --reporters=jest-junit --forceExit",
     "test:ci": "yarn test:cli && yarn test:packages",
     "test:watch": "jest test/ packages/ --watch",
     "test:smoke": "smoketests/smoketests.sh",
-    "publish:monorepo": "yarn format && yarn test && yarn build && lerna publish -m \"chore: monorepo version update\""
+    "publish:monorepo": "yarn test && yarn build && yarn lerna-changelog && lerna publish -m \"chore: monorepo version update\" --canary --preid rc --dist-tag next"
```

Comment:
canary will append a `SHA` to the package version right?

Reply:
Hope yes :smile: I am still testing and watching everything, so this may not be the last commit

Reply:
lerna do not have `--dry` flag to test, so I am afraid of accidentally breaking something

Reply:
Also, does it make sense to run tests here?

We anyways merge on CI green

Reply:
you can try it on your fork first

Reply:
or maybe after npm logout so it won't have access to publish

Reply:
> Also, does it make sense to run tests here?

Honestly, I don't have a strong opinion here, some packages do it, some not, but I agree, let's remove, we have CI and should catch problem there, not on publish command

> or maybe after npm logout so it won't have access to publish

that's exactly what i do :smile: